### ISSUE TITLE
questions : ar.js work in Android webview ,its fps is low,only 8 fps,i donnot how to do 

### ISSUE THREAD

**Issue by webglzhang** [jeromeetienne/AR.js#301]
**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Operating System and Device Name**

**Comment by albertocruz96** [comment]
Hi,

i am developing an app related with ar.js and WebView, but i'am incapable of load the URL in the WebView.
Could you please share your code?

Thanks

**Comment by aj019** [comment]
Yes can you please share how you were able to use ar.js and three.js in your android webview. I am trying to use my android device camera to provide the video stream.

**Comment by webglzhang** [comment]
I've tried two ways to succeed       
The first is to load HTML and JS and resource files locally. You only need to set an attribute of Webview.           
 The second is to get resources through the server. You need to make your server support HTTPS protocol.

**Comment by danieldocumentos** [comment]
No está resuelto, no lo cierren, probé con con una página con certificado SSL y tampoco funciona
